test/functionBuilder.js

```diff
@@ -361,13 +361,24 @@ describe('Function Builder', function () {
       var fn;
       var result;
       var oldCounter;
+      let obj;
       before(function () {
         counter = 0;
         fn = getter();
+
+        obj = {cid: 1};
+        fn.call(obj, 1);
```

Comment:
Oof, yeah I suppose this is fine. I take it our function builder test harness doesn't lend itself well to methods.